packages/account/src/Components/poa/status/unverified/unverified.tsx

```diff
@@ -1,22 +1,25 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import { Button, Icon, Text } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import IconMessageContent from '../../../icon-message-content';
 
 type TUnverified = {
+    title?: ReactNode;
+    description?: ReactNode;
     onClick: () => void;
+    button_text?: ReactNode;
 };
 
-export const Unverified = ({ onClick }: TUnverified) => {
+export const Unverified = ({ title, description, button_text, onClick }: TUnverified) => {
     return (
         <IconMessageContent
-            message={localize('We could not verify your proof of address')}
-            text={localize('Please check your email for details.')}
+            message={title ?? localize('We could not verify your proof of address')}
+            text={description ?? localize('Please check your email for details.')}
```

Comment:
text is optional in IconMessageContent, so maybe no needs to use ?? here?
and for the title, is it expected that title can be empty string?

Reply:
if we pass title, we will show the message , else we will show the default message